portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,8 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String message = "Welcome to my portfolio site! Hello Alisha!";
```

Comment:
This variable here isn't changing. That makes it `final` (meaning it will not/cannot be changed). There's another concept, `static` that can apply here - those are variables that are at the class level and there is only one copy of it no matter how many objects get instantiated.

To be a bit more proper:

`private static final String WELCOME_MESSAGE = "Welcome to my.....";`

This declaration goes right under the class declaration line. Note the variable name, it's java style for static finals.